packages/cdktf/lib/terraform-stack.ts

```diff
@@ -151,3 +146,18 @@ export class TerraformStack extends Construct {
     fs.writeFileSync(resourceOutput, JSON.stringify(this.toTerraform(), undefined, 2));
   }
 }
+
+function terraformElements(node: IConstruct, into: TerraformElement[] = []): TerraformElement[] {
+  if (node instanceof TerraformElement) {
+    into.push(node);
+  }
+
+  for (const child of Node.of(node).children) {
+    // Don't recurse into a substack
+    if (TerraformStack.isStack(child)) { continue; }
```

Comment:
Is this aiming at nested stacks? Do you have an example what this is about?

Reply:
Yeah, this would be for nested stacks. Could just remove since we don't have them. I was just porting some changes from AWS CDK.

Reply:
right ok - well, if it works let's keep it. We'll likely need that for #35